src/frontend/src/styles/pages/pages.scss

```diff
@@ -1 +1,2 @@
-@import '../../pages/Dashboard/Dashboard.scss';
+@import './Dashboard.scss';
+@import './Groups.scss';
```

Comment:
I've moved the style files for pages to the Styles module. This allows us to re-use scss mixins and overrides. I'm also slowly losing my mind looking for scss files everywhere